src/test/features/response/form/models/timelineBrakedown.ts

```diff
@@ -0,0 +1,133 @@
+
+      expectAllRowsToBeEmpty(actual.rows)
+    })
+
+    it('should return TimelineBreakdown with first element on list populated', () => {
+      const actual: TimelineBreakdown = TimelineBreakdown.fromObject({ rows: [{ date: 'May', description: 'OK' }] })
+
+      const populatedItem: TimelineRow = actual.rows.pop()
+
+      expect(populatedItem.date).to.eq('May')
+      expect(populatedItem.description).to.eq('OK')
+
+      expectAllRowsToBeEmpty(actual.rows)
+    })
+
+    it('should return object with list of TimelineRow longer than default', () => {
+      const actual: TimelineBreakdown = TimelineBreakdown.fromObject(
+        {
+          rows: [
+            { date: 'Jan', description: 'OK' },
+            { date: 'Feb', description: 'OK' },
+            { date: 'Mar', description: 'OK' },
+            { date: 'Apr', description: 'OK' },
+            { date: 'May', description: 'OK' },
+            { date: 'Jun', description: 'OK' }
+          ]
+        }
+      )
+
+      expect(actual.rows.length).to.be.greaterThan(INIT_ROW_COUNT)
+      expectAllRowsToBePopulated(actual.rows)
+    })
+  })
+
+  describe('deserialize', () => {
+
+    it('should return valid TimelineBreakdown object with list of empty TimelineRow', () => {
+      const actual: TimelineBreakdown = new TimelineBreakdown().deserialize({})
+
+      expect(actual.rows.length).to.be.eq(INIT_ROW_COUNT)
+      expectAllRowsToBeEmpty(actual.rows)
+    })
+
+    it('should return valid TimelineBreakdown object with populated first TimelineRow', () => {
+      const actual: TimelineBreakdown = new TimelineBreakdown().deserialize(
+        { rows: [{ date: 'May', description: 'OK' }] }
+      )
+
+      expect(actual.rows.length).to.be.eq(INIT_ROW_COUNT)
+
+      const populatedItem: TimelineRow = actual.rows[0]
+
+      expect(populatedItem.date).to.eq('May')
+      expect(populatedItem.description).to.eq('OK')
+
+      expectAllRowsToBeEmpty(actual.rows.slice(1))
+    })
+
+    it('should return valid TimelineBreakdown object with list of row longer than default length', () => {
+      const actual: TimelineBreakdown = new TimelineBreakdown().deserialize(
+        {
+          rows: [
+            { date: 'Jan', description: 'OK' },
+            { date: 'Feb', description: 'OK' },
+            { date: 'Mar', description: 'OK' },
+            { date: 'Apr', description: 'OK' },
+            { date: 'May', description: 'OK' }
+          ]
+        }
+      )
+
+      expectAllRowsToBePopulated(actual.rows)
+    })
+  })
+
+  describe('appendRow', () => {
+
+    it('adds empty element to list of rows', () => {
+      const actual: TimelineBreakdown = new TimelineBreakdown()
+
+      expect(actual.rows.length).to.be.eq(INIT_ROW_COUNT)
+
+      actual.appendRow()
+
+      expect(actual.rows.length).to.be.eq(INIT_ROW_COUNT + 1)
+    })
+  })
+})
+
+function expectAllRowsToBeEmpty (rows: TimelineRow[]) {
+  rows.forEach(item => {
+    expect(item instanceof TimelineRow).to.eq(true)
```

Comment:
You can use [`instanceOf`](http://chaijs.com/api/bdd/#method_instanceof) Chai assertion here.